lib/heroku_rails_deploy/deployer.rb

```diff
@@ -105,6 +133,39 @@ def run_heroku_command(app_name, command)
       run_command(cli_command)
     end
 
+    def registry_url(app_name)
+      result = Bundler.with_clean_env { `heroku config -a #{app_name} | grep AVRO_SCHEMA_REGISTRY_URL:` }
```

Comment:
Should we modify `run_command!` to return process output so we can have common error handling for stuff like this?